rust/src/client.rs

```diff
@@ -235,6 +235,23 @@ pub(crate) fn client_render_download_progress(
     }
 }
 
+pub(crate) fn microdnf_install(arg: &str) -> Result<()> {
+    let mut packages;
+    if !arg.starts_with("install"){
+        return Err(anyhow!("Command '{}' not supported, currently only install is allowed.", arg));
+    } else {
+        packages = arg.replace("install", "");
+        packages = packages.replace(" ", "");
+    }
+    let microdnf_status = Command::new("microdnf")
+        .args(&["install", &packages, "-y"]).status().expect("No package matches");
```

Comment:
What would be the behavior of -y for non container systems? Do we ignore it or we change rpm-ostree to be interactive without it and need user ack before proceeding? 

I see your comment on using it with `-A` but if we wanted 1:1 mapping i.e. `microdnf install -y` = `rpmostree install -y` since we already don't have user interaction after `rpm-ostree install pkg1` not sure what we accomplish for the user experience. If I want to call `rpm-ostree` on my build like I do on my system then not requiring -y and just adding it in the background seems to be a consistent experience for the user.